.github/workflows/push.yaml

```diff
@@ -281,6 +298,13 @@ jobs:
             "${root_dir}"/OBS-Studio-${{ steps.check.outputs.version }}-Ubuntu-x86_64-dbsym.ddeb
           mv -v "${ubuntu_sources_name}/"obs-studio-*-sources.tar.gz \
             "${root_dir}"/OBS-Studio-${{ steps.check.outputs.version }}-Sources.tar.gz
+          mv -v "${windows_installer_name}/"OBS-Studio-*-x64.exe \
+            "${root_dir}"/OBS-Studio-${{ steps.check.outputs.version }}-Windows-x64-Installer.exe
+          echo '::endgroup::'
+          
+          echo '::group::Zipping Windows Artifacts'
```

Comment:
Archiving is actually out of scope for this workflow job, the archives should already be "final" before this job runs and its only task is to rename the artifacts. 

Thus the zip generation should be handled in the job that generates them in the first place.